exchanges/bitfinex/bitfinex.go

```diff
@@ -475,8 +476,12 @@ func (b *Bitfinex) GetCandles(symbol, timeFrame string, start, end, limit int64,
 		return nil, errors.New("no data returned")
 	}
 
+	tempTime := response[0].(float64)
+	sec, dec := math.Modf(tempTime)
+	timestamp := time.Unix(int64(sec), int64(dec*(1e9)))
```

Comment:
Just a quick check I am getting an int64 for the raw response 1591755840000, So I don't know if this is needed? Maybe this: ```time.Unix(int64(response[i][0].(float64)/1000), 0)``` will suffice?